<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администрирование съемных МНИ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Учет МНИ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tvMni">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Контроль использования МНИ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tvReport"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>СВТ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableView" name="tvSVT">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Сотрудники</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="tvPerson">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Подразделения</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tvDivision">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actAddMni">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/add_mni</normaloff>:/icons/add_mni</iconset>
   </property>
   <property name="text">
    <string>Добавить МНИ</string>
   </property>
   <property name="toolTip">
    <string>Добавить вновь зарегистрированный МНИ МНИ</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actAddDivision">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/add_group</normaloff>:/icons/add_group</iconset>
   </property>
   <property name="text">
    <string>Добавить подразделение</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actDelDivision">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/del_group</normaloff>:/icons/del_group</iconset>
   </property>
   <property name="text">
    <string>Удалить подразделение</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actEditDivision">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/edit_group</normaloff>:/icons/edit_group</iconset>
   </property>
   <property name="text">
    <string>Редактировать подразделение</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actAddSVT">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/add_svt</normaloff>:/icons/add_svt</iconset>
   </property>
   <property name="text">
    <string>Добавить СВТ</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actDelSVT">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/del_svt</normaloff>:/icons/del_svt</iconset>
   </property>
   <property name="text">
    <string>Удалить СВТ</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actEditSVT">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/edit_svt</normaloff>:/icons/edit_svt</iconset>
   </property>
   <property name="text">
    <string>Редактировать СВТ</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actAddPerson">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/add_user</normaloff>:/icons/add_user</iconset>
   </property>
   <property name="text">
    <string>Добавить ответственного</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actDelPerson">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/del_user</normaloff>:/icons/del_user</iconset>
   </property>
   <property name="text">
    <string>Удалить ответственного</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actEditPerson">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/edit_user</normaloff>:/icons/edit_user</iconset>
   </property>
   <property name="text">
    <string>Редактировать ответственного</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actRefreshMNI">
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actDelMni">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/del_mni</normaloff>:/icons/del_mni</iconset>
   </property>
   <property name="text">
    <string>Удалить МНИ</string>
   </property>
   <property name="toolTip">
    <string>Удалить МНИ</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actEditMni">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/edit_mni</normaloff>:/icons/edit_mni</iconset>
   </property>
   <property name="text">
    <string>Редактировать МНИ</string>
   </property>
   <property name="toolTip">
    <string>Редактировать МНИ</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actDeployRules">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/deploy-one</normaloff>:/icons/deploy-one</iconset>
   </property>
   <property name="text">
    <string>Применить разрешения</string>
   </property>
   <property name="toolTip">
    <string>Формирует список правил и отправляет их на контролируемое СВТ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actDeployAll">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/deploy-all</normaloff>:/icons/deploy-all</iconset>
   </property>
   <property name="text">
    <string>Применить ко всем ...</string>
   </property>
   <property name="toolTip">
    <string>Формирует список правил и отправляет их на все контролируемые СВТ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="flash-admin.qrc"/>
 </resources>
 <connections/>
</ui>
